spotify/scripts/dscc-scripts.js

```diff
@@ -0,0 +1,79 @@
+const open = require('opn');
+const argparse = require('argparse');
+const packageJson = require('../package.json');
+const cp = require('child_process');
+const chalk = require('chalk');
+const terminalLink = require('terminal-link');
+
+const exec = (command, options) => {
+  return new Promise((resolve, reject) => {
+    const child = cp.exec(command, options, (err, stdout, stderr) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve({out: stdout, err: stderr});
+      }
+    });
+    child.stdout.pipe(process.stdout);
+    child.stderr.pipe(process.stderr);
+  });
+};
+
+const parser = new argparse.ArgumentParser({
+  version: '1.0',
+  addHelp: true,
+  description: 'Scripts for managing connectors.'
+});
+
+parser.addArgument(['-s', '--script'], {
+  choices: ['try_production', 'update_production', 'try_latest'],
+  dest: 'script',
```

Comment:
I think `dest` is optional, it picks it out from `--script` if it uses the same behavior as Python argparse